FEM/matrix_class.cpp

```diff
@@ -1675,8 +1675,6 @@ void CSparseMatrix::Diagonize(const long idiag, const double b_given, double* b)
         /// Diagonal entry and the row where the diagonal entry exists
         j = diag_entry[id];
         vdiag = entry[(ii * DOF + ii) * size_entry_column + j];
-        if (fabs(vdiag) < DBL_EPSILON)
```

Comment:
It was added in one of my previous PRs. However it makes the solver divergences for some problems with subdomain deactivation. Therefore, theses changes are dropped now.